sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>Azure Event Hubs is a highly scalable publish-subscribe service that can ingest millions of events per second and stream them to multiple consumers.  This client library allows for both publishing and consuming events using Azure Event Hubs. For more information about Event Hubs, see https://azure.microsoft.com/en-us/services/event-hubs/</Description>
-    <Version>5.0.1</Version>
+    <Version>5.1.0-preview.1</Version>
```

Comment:
Is this right? We are making it 5.0.1 everywhere else, including in the Processor csproj.

Reply:
This is the "post release bump" to the next available version.   That way, any unintentional publishes are done as a preview and do not confuse folks by seeming to be legit full releases.    There will be a similar change to the processor once the package is published.